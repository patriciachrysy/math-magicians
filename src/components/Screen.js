class Screen extends React.Component {
    constructor(props) {
        super(props);
    }

    render = () => {
        return (
            <div class="screen-frame">
                <h1>{this.props.value}</h1>
            </div>
        )
    }
}