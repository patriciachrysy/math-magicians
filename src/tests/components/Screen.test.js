import React from 'react';
import renderer from 'react-test-renderer';

import { render, screen } from '@testing-library/react';
import Screen from '../../components/Screen';

it('renders correctly', () => {
  const tree = renderer.create(<Screen value="0" />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Render Screen component', () => {
  render(<Screen value="0" />);
  screen.debug();
});
