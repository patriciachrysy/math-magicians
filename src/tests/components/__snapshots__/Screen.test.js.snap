// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="screen-frame"
>
  <span>
    0
  </span>
</div>
`;
